default.hbs

```diff
@@ -2,11 +2,20 @@
 <html lang="{{@site.lang}}">
 
 <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>{{meta_title}}</title>
-    <link rel="stylesheet" href="{{asset "built/screen.css"}}">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Muli:400,400i,700,700i,800">
+    <link rel="stylesheet" href="{{asset "built/screen.css"}}" />
+
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+
+    <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;700&family=Muli:ital,wght@0,400;0,700;0,800;1,400;1,700&display=swap" />
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=JetBrains+Mono:wght@400;700&family=Muli:ital,wght@0,400;0,700;0,800;1,400;1,700&display=swap" media="print" onload="this.media='all'" />
```

Comment:
We shouldn't bulk the Google Fonts stack with another font.

Reply:
I understand that you want to keep your design minimal. But the behavior is a bit strange with its PRE and CODE tags. When the theme is light, light background colors should be shown and not dark and vice versa.

If you don't want to implement a monospace font, this is the right thing to do.

Otherwise, it is a good idea to implement a monospace font for these types of tags so that they are consistent across browsers and operating systems.
![dawn-code-exam1](https://user-images.githubusercontent.com/14000242/103982965-149a0500-514a-11eb-8b33-305bc559097e.png)
![dawn-code-exam2](https://user-images.githubusercontent.com/14000242/103982969-1794f580-514a-11eb-935b-b9d6d67753c7.png)
![dawn-code-exam3](https://user-images.githubusercontent.com/14000242/103983049-401cef80-514a-11eb-9a46-6b4178eaa8ee.png)
![dawn-code-exam4](https://user-images.githubusercontent.com/14000242/103983055-4317e000-514a-11eb-899c-a3af67ab310a.png)

I wish you a good day.

Reply:
I agree with your points. In this case:

- We should use system mono font stack for this instead of external one. Reference: https://systemfontstack.com/
- Reverse the background color to make it suit better for light/dark mode.

Just pushed a fix for this. https://github.com/TryGhost/Dawn/commit/7e34b824218162244691b7718fd22d8e0ea4c5b3

Here is how it looks now on light/dark mode.

![Screen Shot 2021-01-08 at 3 44 15 PM](https://user-images.githubusercontent.com/1418797/103988318-d7c12300-51c8-11eb-89ab-b6c6091afd6c.png)
![Screen Shot 2021-01-08 at 3 44 31 PM](https://user-images.githubusercontent.com/1418797/103988324-da237d00-51c8-11eb-9a03-9cdab640f074.png)